hack/bootstrap-cluster.sh

```diff
@@ -67,3 +67,5 @@ echo "OK"
 echo
 echo "Login/password uses your OpenShift credentials ('Login with OpenShift' button)"
 echo
+
+oc project default
```

Comment:
not needed.

Reply:
You're right, I'll remove it. The motivation was to avoid landing on tekton-chains, which isn't happening any more.

Reply:
It's removed in the latest revision.